tests/include/mir/test/doubles/mock_gl_rendering_provider.h

```diff
@@ -21,6 +21,7 @@
 
 #include "mir/graphics/gl_config.h"
 #include "mir/graphics/platform.h"
+#include "mir/renderer/gl/gl_surface.h"
```

Comment:
Spurious addition?

Reply:
Apparently not

Reply:
Yeah, need the definition so the destructor can be instantiated!